cli/utilities/packages.py

```diff
@@ -201,6 +202,28 @@ def register(self, username, password, serverurl=None, baseurl=None, force=False
                 "Failed to register node(s) to subscription manager"
             )
 
+    def clean(self):
+        """
+        Cleans all local subscription data: entitlements, certs, identity.
+
+        Raises:
+            SubscriptionManagerError: if clean fails.
+        """
+        self.execute(sudo=True, long_running=True, cmd=f"{self.base_cmd} clean")
+
+    def cleanup_subscription_files(self):
+        """
+        Remove residual subscription-related certificate and identity files.
+
+        This method removes files that are not cleaned by `subscription-manager clean`,
+        such as consumer and product certificates.
+
+        Raises:
+            SubscriptionManagerError: if the file cleanup command fails.
+        """
+        cmd = "rm -rf /etc/pki/consumer /etc/pki/product"
+        self.execute(sudo=True, long_running=True, cmd=cmd)
```

Comment:
Please using a blocking call... i.e. set `long_running=False`

Reply:
Updated the long_running flag to False to ensure it's a blocking call